core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -773,20 +781,20 @@ public Variation getVariation(@Nonnull Experiment experiment,
     public Variation getVariation(@Nonnull Experiment experiment,
                                   @Nonnull String userId,
                                   @Nonnull Map<String, ?> attributes) throws UnknownExperimentException {
-        return getVariation(experiment, userId, attributes, getProjectConfig());
+        return getVariation(getProjectConfig(), experiment, userId, attributes);
     }
 
     @Nullable
-    public Variation getVariation(@Nonnull Experiment experiment,
-                                  @Nonnull String userId,
-                                  @Nonnull Map<String, ?> attributes,
-                                  @Nonnull ProjectConfig projectConfig) throws UnknownExperimentException {
+    private Variation getVariation(@Nonnull ProjectConfig projectConfig,
```

Comment:
Making this private as public methods are not supposed to take `projectConfig`

Reply:
Technically backwards incompatible, but I'm fine with it.

Reply:
Backwards incompatible because it was public in an alpha release? I think we should be comfortable removing this right away.